suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -726,16 +734,17 @@ export const isAccountOutdated = (account: Account, freshInfo: AccountInfo) => {
         case 'ripple':
             // different sequence or balance
             return (
+                isDifferentBalance() ||
                 freshInfo.misc!.sequence !== account.misc.sequence ||
-                freshInfo.balance !== account.balance ||
                 freshInfo.misc!.reserve !== account.misc.reserve
             );
         case 'ethereum':
             return (
+                isDifferentBalance() ||
                 freshInfo.misc!.nonce !== account.misc.nonce ||
-                freshInfo.balance !== account.balance || // balance can change because of beacon chain txs (staking) |
                 JSON.stringify(freshInfo?.misc?.stakingPools) !==
-                    JSON.stringify(account?.misc?.stakingPools)
+                    JSON.stringify(account?.misc?.stakingPools) ||
+                areDifferentTokens()
```

Comment:
total amount of txs changes when you receive tokens so no changes here are needed